planning/motion_velocity_smoother/src/smoother/analytical_jerk_constrained_smoother/analytical_jerk_constrained_smoother.cpp

```diff
@@ -291,19 +295,17 @@ boost::optional<TrajectoryPoints> AnalyticalJerkConstrainedSmoother::applyLatera
   for (double s = 0; s < in_arclength.back(); s += points_interval) {
     out_arclength.push_back(s);
   }
-  auto output = trajectory_utils::applyLinearInterpolation(in_arclength, input, out_arclength);
-  if (!output) {
-    RCLCPP_WARN(logger_, "Interpolation failed at lateral acceleration filter.");
-    return boost::none;
```

Comment:
@purewater0901 
I think case output none is necesary.

Reply:
New resampling functions from motion utils do not use boost::optional, therefore, we do not need to handle none output